.gcloudignore

```diff
@@ -0,0 +1,3 @@
+.git
```

Comment:
nit (non-blocking): What's this file for?

Reply:
To prevent uploading the entire Git repository to Google Cloud Build.  This line is standard in a .gcloudignore file because when you're building, you don't care about the git history. Our file matches the official example: https://cloud.google.com/sdk/gcloud/reference/topic/gcloudignore

I believe that prom-bench would be utilizing this. However, the original change predates our use of prom-bench: https://github.com/GoogleCloudPlatform/prometheus/commit/623141cf6e2b9b6aad2e1a52ad1f6cb7152c9e9c

Reply:
Oh interesting. Ok - the git blame was confusing here. I'm assuming we technically don't need that anymore. It was probably added for doing manual cloud builds. But happy to keep for now, given precedent.